pkgs/development/python-modules/scholarly/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, unittestCheckHook
+
+, arrow
+, beautifulsoup4
+, bibtexparser
+, deprecated
+, fake-useragent
+, free-proxy
+, httpx
+, python-dotenv
+, selenium
+, sphinx-rtd-theme
+, typing-extensions
+}:
+
+buildPythonPackage rec {
+  pname = "scholarly";
+  version = "1.7.11";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-LJg91E2dk5im8mBRAq5uVYYCO0HruuwUYZF+5I6xU/A=";
+  };
+
+  propagatedBuildInputs = [
+    arrow
+    beautifulsoup4
+    bibtexparser
+    deprecated
+    fake-useragent
+    free-proxy
+    httpx
```

Comment:
Does this work? I though it was needed to specify they are from the python package set.

Reply:
This should work. It does for me locally, though. I think with the `buildPythonPackage` function, specifying that the packages are from the python package set is not necessary. Also see other python packages.